### PR TITLE
Added Latitude and Longitude values

### PR DESCRIPTION
**Pull request by joshkopecek**
I wrote this addition for a project which involves a lot of GeoJSON and I kept repeatedly needing to generate random numbers within the GeoJSON standard. Happy to field questions and alterations :-)

### CODE REVIEW

File: spec/random-data-spec.coffee [hunk 1]
```diff
@@ -279,3 +279,9 @@ describe "Random Data", ->
   it "inserts random tv", ->
     spyOn(chance, 'tv').andReturn('KICE')
     dataTest 'tv', 'KICE'
+  it "inserts random longitude", ->
+    spyOn(chance, 'longitude').andReturn('-49.273096')
```
Comment (RichardSlater) [inline spec/random-data-spec.coffee]: I think `spyOn(change, 'longitude')` should probably be `spyOn(change, 'float')`

File: spec/random-data-spec.coffee [hunk 2]
```diff
@@ -279,3 +279,9 @@ describe "Random Data", ->
...
+    dataTest 'longitude', '-49.273096'
+  it "inserts random latitude", ->
+    spyOn(chance, 'latitude').andReturn('-85.954858')
```
Comment (RichardSlater) [inline spec/random-data-spec.coffee]: I think `spyOn(change, 'latitude')` should probably be `spyOn(change, 'float')`

**Comment by RichardSlater** [comment]
Hi - thank you so much for the contribution, I think the tests need a little tweak as chance doesn't have an underlying `latitude` or `longitude` function you need to spy on `float` instead.

**Comment by joshkopecek** [comment]
Yes you're absolutely right, sorry I wasn't sure how to run the tests !? Could you give me a heads up?

**Comment by RichardSlater** [comment]
Of course, it is the least I could do.  There are two ways to run the tests from your workstation, the first is within Atom press <kbd>Ctrl</kbd>-<kbd>Alt</kbd>-<kbd>P</kbd> this will bring up the built-in [Jamine 1.3](http://jasmine.github.io/1.3/introduction.html) test runner.  The second way, from a terminal, is to change directory to the location of the project and run:

```
apm test
```

This will run Jasmine from the command line and feedback to the console.

**Comment by joshkopecek** [comment]
@RichardSlater `floating` worked. That was a quick fix. Thanks for the help with testing!

**Comment by RichardSlater** [comment]
Thank you again for your contribution Josh, this is all packaged up and on the [Atom Package Manager](https://atom.io/packages/random) site now.

**Comment by joshkopecek** [comment]
Thanks @RichardSlater happy to have contributed!

**Pull request merged**